01-introduction.Rmd

```diff
@@ -94,7 +93,7 @@ Of course, (geocomputational) algorithms are powerful tools that can become high
 However, all algorithms are composed of smaller parts.
 By teaching you its foundations and underlying structure, we aim to empower you to create your own innovative solutions to geographic data problems.
 
-## Why geocomputation with R?
+## Why use geocomputation with R?
```

Comment:
I disagree - we don't use geocomputation with R. We use R for geocomputation. The previous heading was better. I suggest reverting to that or changing it to:

```
Why use R for geocomputation?
```

Reply:
I guess the heading was written in accordance with our book title but I agree with you @Robinlovelace that this might be misleading here, and would also opt for `Why use R for geocomputation?`

Reply:
Thanks, done - https://github.com/Robinlovelace/geocompr/pull/338/commits/84c68a3d17009114d4d76bdd5f75a3ff95893d85